### PR TITLE
[codestyle] admin non-components xml files

### PR DESCRIPTION
**Pull request by brianteeman**
Review all the xml files in the administrator/components folder and make sure they are formatted according to the coding standards

(I had a long train journey)

### CODE REVIEW

File: administrator/modules/mod_status/mod_status.xml [hunk 1]
```diff
@@ -20,70 +20,77 @@
 	<help key="JHELP_EXTENSIONS_MODULE_MANAGER_ADMIN_STATUS" />
 	<config>
 		<fields name="params">
-			<fieldset name="basic">
+		<fieldset name="basic">
```
Comment (zero-24) [inline administrator/modules/mod_status/mod_status.xml]: this whole fieldset block needs one more tab

File: administrator/modules/mod_custom/mod_custom.xml [hunk 2]
```diff
@@ -26,11 +26,12 @@
 			<fieldset name="options" label="COM_MODULES_BASIC_FIELDSET_LABEL">
 				<field
 					name="prepare_content"
-					type="radio"
 					label="MOD_CUSTOM_FIELD_PREPARE_CONTENT_LABEL"
-					class="btn-group btn-group-yesno"
 					description="MOD_CUSTOM_FIELD_PREPARE_CONTENT_DESC"
-					default="1">
+					type="radio"
```
Comment (wojsmol) [inline administrator/modules/mod_custom/mod_custom.xml]: type before lebal

File: administrator/modules/mod_feed/mod_feed.xml [hunk 3]
```diff
@@ -25,59 +25,57 @@
...
+					label="MOD_FEED_FIELD_DESCRIPTION_LABEL"
+					description="MOD_FEED_FIELD_DESCRIPTION_DESC"
 					class="btn-group btn-group-yesno"
 					default="1"
-					label="MOD_FEED_FIELD_DESCRIPTION_LABEL"
-					description="MOD_FEED_FIELD_DESCRIPTION_DESC">
-					<option
-						value="1">JYES</option>
-					<option
-						value="0">JNO</option>
```
Comment (wojsmol) [inline administrator/modules/mod_feed/mod_feed.xml]: options deleted

File: administrator/modules/mod_popular/mod_popular.xml [hunk 4]
```diff
@@ -54,31 +58,34 @@
...
+				/>
 
 				<field
 					name="moduleclass_sfx"
-					type="textarea" rows="3"
+					type="textarea"
 					label="COM_MODULES_FIELD_MODULECLASS_SFX_LABEL"
-					description="COM_MODULES_FIELD_MODULECLASS_SFX_DESC" />
+					description="COM_MODULES_FIELD_MODULECLASS_SFX_DESC"
+					 rows="3"
```
Comment (wojsmol) [inline administrator/modules/mod_popular/mod_popular.xml]: unnidet soace

File: administrator/modules/mod_stats_admin/mod_stats_admin.xml [hunk 5]
```diff
@@ -92,15 +95,16 @@
 				<field
 					name="cache_time"
 					type="text"
-					default="900"
 					label="COM_MODULES_FIELD_CACHE_TIME_LABEL"
-					description="COM_MODULES_FIELD_CACHE_TIME_DESC" />
+					description="COM_MODULES_FIELD_CACHE_TIME_DESC"
+					default="900"
+				/>
 				<field
```
Comment (wojsmol) [inline administrator/modules/mod_stats_admin/mod_stats_admin.xml]: empty line

File: administrator/modules/mod_status/mod_status.xml [hunk 6]
```diff
@@ -22,68 +22,75 @@
...
+				type="radio"
+				label="MOD_STATUS_FIELD_SHOW_VIEWADMIN_LABEL"
+				description="MOD_STATUS_FIELD_SHOW_VIEWADMIN_DESC"
+				class="btn-group btn-group-yesno"
+				default="0"
+				>
+				<option value="1">JSHOW</option>
+				<option value="0">JHIDE</option>
+			</field>
+			<field
```
Comment (wojsmol) [inline administrator/modules/mod_status/mod_status.xml]: all fields in this fieldset add empy line

File: administrator/templates/isis/templateDetails.xml [hunk 7]
```diff
@@ -47,84 +47,124 @@
...
-				<field name="sidebarColor" class="" type="color" default="#0088CC"
-				validate="color"
-				label="TPL_ISIS_COLOR_SIDEBAR_LABEL"
-				description="TPL_ISIS_COLOR_SIDEBAR_DESC" />
+				<field 
+					name="sidebarColor" 
+					label="TPL_ISIS_COLOR_SIDEBAR_LABEL"
+					description="TPL_ISIS_COLOR_SIDEBAR_DESC" 
+					class="" 
+					type="color"
```
Comment (wojsmol) [inline administrator/templates/isis/templateDetails.xml]: type before lebal

File: administrator/templates/isis/templateDetails.xml [hunk 8]
```diff
@@ -47,84 +47,124 @@
...
 
-				<field name="loginBackgroundColor" class="" type="color" default="#17568C"
-				validate="color"
-				label="TPL_ISIS_COLOR_LOGIN_BACKGROUND_LABEL"
-				description="TPL_ISIS_COLOR_LOGIN_BACKGROUND_DESC" />
+				<field 
+					name="loginBackgroundColor" 
+					label="TPL_ISIS_COLOR_LOGIN_BACKGROUND_LABEL"
+					description="TPL_ISIS_COLOR_LOGIN_BACKGROUND_DESC"
+					type="color"
```
Comment (wojsmol) [inline administrator/templates/isis/templateDetails.xml]: type before lebal

File: administrator/templates/isis/templateDetails.xml [hunk 9]
```diff
@@ -47,84 +47,124 @@
...
+				 />
 
-				<field name="loginLogoFile" class="" type="media" default=""
-				label="TPL_ISIS_LOGIN_LOGO_LABEL"
-				description="TPL_ISIS_LOGIN_LOGO_DESC" />
+				<field 
+					name="loginLogoFile" 
+					label="TPL_ISIS_LOGIN_LOGO_LABEL"
+					description="TPL_ISIS_LOGIN_LOGO_DESC"
+					type="media"
```
Comment (wojsmol) [inline administrator/templates/isis/templateDetails.xml]: type before lebal

File: administrator/templates/isis/templateDetails.xml [hunk 10]
```diff
@@ -47,84 +47,124 @@
...
-				<option
-					value="0">JNO</option>
+				<field 
+					name="displayHeader" 
+					type="radio"
+					label="TPL_ISIS_HEADER_LABEL"
+					description="TPL_ISIS_HEADER_DESC"
+					class="btn-group btn-group-yesno"
+					default="1"
+					filter="integer">
```
Comment (wojsmol) [inline administrator/templates/isis/templateDetails.xml]: > in new line

**Comment by wojsmol** [comment]
@brianteeman check options formating https://developer.joomla.org/coding-standards/xml.html

File: administrator/modules/mod_feed/mod_feed.xml (hunk 3 above)
Comment (andrepereiradasilva) [inline administrator/modules/mod_feed/mod_feed.xml]: @wojsmol they are ok now

File: administrator/modules/mod_latest/mod_latest.xml [hunk 11]
```diff
@@ -25,38 +25,43 @@
...
-					default="c_dsc"
 					label="MOD_LATEST_FIELD_ORDERING_LABEL"
-					description="MOD_LATEST_FIELD_ORDERING_DESC">
+					description="MOD_LATEST_FIELD_ORDERING_DESC"
+					default="c_dsc"
+					>
 					<option
 						value="c_dsc">MOD_LATEST_FIELD_VALUE_ORDERING_ADDED</option>
 					<option
 						value="m_dsc">MOD_LATEST_FIELD_VALUE_ORDERING_MODIFIED</option>
```
Comment (andrepereiradasilva) [inline administrator/modules/mod_latest/mod_latest.xml]: proposed improvement: option in a single line

File: administrator/modules/mod_latest/mod_latest.xml [hunk 12]
```diff
@@ -25,38 +25,43 @@
...
 					name="user_id"
 					type="list"
-					default="0"
 					label="MOD_LATEST_FIELD_AUTHORS_LABEL"
-					description="MOD_LATEST_FIELD_AUTHORS_DESC">
+					description="MOD_LATEST_FIELD_AUTHORS_DESC"
+					default="0"
+					>
 					<option
 						value="0">MOD_LATEST_FIELD_VALUE_AUTHORS_ANYONE</option>
```
Comment (andrepereiradasilva) [inline administrator/modules/mod_latest/mod_latest.xml]: proposed improvement: option in a single line

File: administrator/modules/mod_latest/mod_latest.xml [hunk 13]
```diff
@@ -71,25 +76,26 @@
 					name="layout"
 					type="modulelayout"
 					label="JFIELD_ALT_LAYOUT_LABEL"
-					description="JFIELD_ALT_MODULE_LAYOUT_DESC" />
+					description="JFIELD_ALT_MODULE_LAYOUT_DESC" 
+				/>
 
 				<field
 					name="moduleclass_sfx"
 					type="textarea" rows="3"
```
Comment (andrepereiradasilva) [inline administrator/modules/mod_latest/mod_latest.xml]: rows goes down

File: administrator/modules/mod_logged/mod_logged.xml [hunk 14]
```diff
@@ -26,14 +26,16 @@
 					type="text"
 					default="5"
```
Comment (andrepereiradasilva) [inline administrator/modules/mod_logged/mod_logged.xml]: default goes down

File: administrator/modules/mod_popular/mod_popular.xml [hunk 15]
```diff
@@ -25,27 +25,31 @@
...
 					name="user_id"
 					type="list"
-					default="0"
 					label="MOD_POPULAR_FIELD_AUTHORS_LABEL"
-					description="MOD_POPULAR_FIELD_AUTHORS_DESC">
+					description="MOD_POPULAR_FIELD_AUTHORS_DESC"
+					default="0"
+					>
 					<option
 						value="0">MOD_POPULAR_FIELD_VALUE_ANYONE</option>
```
Comment (andrepereiradasilva) [inline administrator/modules/mod_popular/mod_popular.xml]: proposed improvement: option in a single line

File: administrator/modules/mod_popular/mod_popular.xml [hunk 16]
```diff
@@ -54,31 +58,34 @@
 						value="not_me">MOD_POPULAR_FIELD_VALUE_NOT_ADDED_OR_MODIFIED_BY_ME</option>
 				</field>
 			</fieldset>
+
 			<fieldset
 				name="advanced">
```
Comment (andrepereiradasilva) [inline administrator/modules/mod_popular/mod_popular.xml]: single line?

File: administrator/modules/mod_popular/mod_popular.xml (hunk 4 above)
Comment (andrepereiradasilva) [inline administrator/modules/mod_popular/mod_popular.xml]: agree

File: administrator/modules/mod_stats_admin/mod_stats_admin.xml [hunk 17]
```diff
@@ -26,64 +26,67 @@
...
 					type="text"
-					default="0"
 					label="MOD_STATS_FIELD_INCREASECOUNTER_LABEL"
-					description="MOD_STATS_FIELD_INCREASECOUNTER_DESC" />
+					description="MOD_STATS_FIELD_INCREASECOUNTER_DESC"
+					default="0"
+				 />
 			</fieldset>
 			<fieldset
 				name="advanced">
```
Comment (andrepereiradasilva) [inline administrator/modules/mod_stats_admin/mod_stats_admin.xml]: single line?

File: administrator/modules/mod_stats_admin/mod_stats_admin.xml [hunk 18]
```diff
@@ -26,64 +26,67 @@
...
 					name="cache"
 					type="list"
-					default="1"
 					label="COM_MODULES_FIELD_CACHING_LABEL"
-					description="COM_MODULES_FIELD_CACHING_DESC">
+					description="COM_MODULES_FIELD_CACHING_DESC"
+					default="1"
+					>
 					<option
 						value="1">JGLOBAL_USE_GLOBAL</option>
```
Comment (andrepereiradasilva) [inline administrator/modules/mod_stats_admin/mod_stats_admin.xml]: proposed improvement: option in a single line

File: administrator/templates/isis/templateDetails.xml (hunk 7 above)
Comment (andrepereiradasilva) [inline administrator/templates/isis/templateDetails.xml]: yes. name, type, label, description as the others

File: administrator/templates/isis/templateDetails.xml (hunk 8 above)
Comment (andrepereiradasilva) [inline administrator/templates/isis/templateDetails.xml]: yes. name, type, label, description as the others

File: administrator/templates/isis/templateDetails.xml (hunk 9 above)
Comment (andrepereiradasilva) [inline administrator/templates/isis/templateDetails.xml]: yes. name, type, label, description as the others

File: administrator/templates/isis/templateDetails.xml (hunk 10 above)
Comment (andrepereiradasilva) [inline administrator/templates/isis/templateDetails.xml]: agree, as the others

File: administrator/templates/isis/templateDetails.xml [hunk 19]
```diff
@@ -47,84 +47,124 @@
...
+					label="TPL_ISIS_STATUS_LABEL"
+					description="TPL_ISIS_STATUS_DESC"
+					default="1"
+					filter="integer"
+					>
+					<option
 					value="1">TPL_ISIS_STATUS_BOTTOM</option>
-				<option
+					<option
 					value="0">TPL_ISIS_STATUS_TOP</option>
```
Comment (andrepereiradasilva) [inline administrator/templates/isis/templateDetails.xml]: proposed improvement: option in a single line

**Review by andrepereiradasilva — COMMENTED** [review]
some minor changes and also agree with @wojsmol comments.
if corrected is fine

File: administrator/modules/mod_feed/mod_feed.xml (hunk 3 above)
Comment (wojsmol) [inline administrator/modules/mod_feed/mod_feed.xml]: @brianteeman corected this in https://github.com/joomla/joomla-cms/pull/15922/commits/8b2075d7c2e4831d65606919a37762d52e3315a7.

**Comment by wojsmol** [comment]
I have tested this item :white_check_mark: successfully on dfddfd5139a6dab2be774cdde6f96c6e6d4c36c0<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/15922">issues.joomla.org/tracker/joomla-cms/15922</a>.</sub>

**Comment by andrepereiradasilva** [comment]
I have tested this item :white_check_mark: successfully on dfddfd5139a6dab2be774cdde6f96c6e6d4c36c0<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/15922">issues.joomla.org/tracker/joomla-cms/15922</a>.</sub>

**Comment by franz-wohlkoenig** [comment]
RTC after two successful tests.

**Comment by brianteeman** [comment]
Thanks

**Pull request merged**